pyretailscience/style/tailwind.py

```diff
@@ -303,19 +303,21 @@
 }
 
 
-def get_color_list(name: str) -> list[str]:
-    """Returns a list of colors from the Tailwind color pallete of the given name.
+def get_color_list(name: str, starting_color_code: int = 200, ending_color_code: int = 800) -> list[str]:
```

Comment:
For this function can you make the defaults the full range of colors. I think that would be more expected. I think it's fine to leave the defaults for the other function at 200 to 800 since that makes it more usable.

```suggestion
def get_color_list(name: str, starting_color_code: int = 50, ending_color_code: int = 950) -> list[str]:
```